src/widget/analog-scale/analog-scalepicker.js

```diff
@@ -133,12 +133,14 @@ class AnalogScaleWidget extends RangeWidget {
     get props() {
         const props = this._props;
         props.touch = support.touch;
+        props.showScale = props.appearances.includes( 'show-scale' );
         props.min = isNumber( this.element.getAttribute( 'min' ) ) ? this.element.getAttribute( 'min' ) : 0;
         props.max = isNumber( this.element.getAttribute( 'max' ) ) ? this.element.getAttribute( 'max' ) : 100;
-        props.step = isNumber( this.element.getAttribute( 'step' ) ) ? this.element.getAttribute( 'step' ) : 1; //( props.type === 'decimal' ? 0.1 : 1 );
+        props.step = isNumber( this.element.getAttribute( 'step' ) ) ? this.element.getAttribute( 'step' ) : ( props.showScale ? 10 : 1 ); //( props.type === 'decimal' ? 0.1 : 1 );
         props.vertical = !props.appearances.includes( 'horizontal' );
         props.ticks = !props.appearances.includes( 'no-ticks' );
         props.maxTicks = 10;
+        props.distress = props.showScale;
```

Comment:
Ah I get it now. This functionality was more or less there already! :D Great you found that! 

I think we just have to change the name of this property (so also in the range widget) to make it more descriptive. Here is a test form that include the distress picker and the regular range widget to test for regressions: https://docs.google.com/spreadsheets/d/1KLQiQyQ5BlN_wd-83p8Eb6xIU-TZp-yvzgrm20HecRI/edit#gid=0. And here it is live: https://enke.to/widgets

How about `showScale` as you've already done?

Reply:
Got it. Will do that.